src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java

```diff
@@ -51,7 +50,7 @@ public synchronized void clearCurrentCommand() {
     }
 
     public synchronized void runCurrentCommand() {
-        currentCommand.execute(driverManager.getCurrentDriver());
+        currentCommand.execute(driverManager.getDriverToDraw());
```

Comment:
changing default API is troublesome for others. It would be better if getCurrentDriver() could stay here.